modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java

```diff
@@ -144,6 +149,37 @@ public static HttpResponse sendPOSTClaimMessage(HttpResponse response, String co
         return httpClient.execute(post);
     }
 
+    /**
+     * To send the response to jit post authentication handler.
+     * @param response Relevant response.
```

Comment:
Format the doc comment. There should be a new line after the header. Also, it'd be great if we can have all comments as single complete sentences.